src/httpclient.h

```diff
@@ -2,6 +2,7 @@
 #define HTTPCLIENT_H_
 
 #include <curl/curl.h>
+#include <sys/socket.h>
```

Comment:
Do we really need this include?

Reply:
This is covered this in the pull request description.

I admit I don't have a better solution for now...